.clang-tidy

```diff
@@ -38,30 +36,47 @@ Checks: >
     readability-*,
     -modernize-avoid-c-arrays,
     -modernize-use-bool-literals,
-    -modernize-use-auto
+    -modernize-use-auto,
+    -modernize-use-trailing-return-type,
+    -readability-magic-numbers,
```

Comment:
Do we use "magic numbers" in many places? I like this check actually

Reply:
In general it's pretty good, but picks up a few bits of test data and some numbers in expressions (masking expressions, etc) which don't really warrant a whole separate constant.  It would be nice if it was a tiny bit less sensitive, so we may want to revisit in the future.

Reply:
I see, thanks